_qs/200-workspace.md

```diff
@@ -13,9 +13,9 @@ Before you start this section, make sure you've checked the [prerequisites](100-
 
 ## The Workspace
 
-A workspace is a directory that has a specific layout so that bnd(tools) knows how to treat your projects. Since we will work in Eclipse we actually have 2 workspaces. The Eclipse workspace is a directory where Eclipse keeps its metadata and we keep that separated from the bnd(tools) workspace.
+A workspace is a directory that has a specific layout so that bnd(tools) knows how to treat your projects. Since we will work in Eclipse we actually have 2 workspaces. The Eclipse workspace is a directory where Eclipse keeps its metadata (a standard Eclipse workspace). We keep that separated from the bnd(tools) workspace which is a standard git repository, though we won't call it that. You'll see why soon.
 
-Since a bnd workspace for enRoute has some specific setup, we clone the workspace from a Github project, [osgi/workspace](https://github.com/osgi/workspace). 
+Since a bnd(tools) workspace for enRoute has some specific setup, we clone the workspace from a Github project, [osgi/workspace](https://github.com/osgi/workspace). 
```

Comment:
bnd workspace is a better term. bndtools means the eclipse IDE front-end to bnd. The bnd workspace exists without bndtools usage (see maven users for example.) We should consistent use bnd workspace rather than bnd(tools) workspace.